src/components/with-datasets/redux/saga.ts

```diff
@@ -1,29 +1,55 @@
-import axios from 'axios';
 import { all, call, put, takeLatest } from 'redux-saga/effects';
 import { authService } from '../../../services/auth/auth-service';
 
-import { getConfig } from '../../../config';
-
 import { SEARCH_DATASETS_REQUESTED } from './actions-types';
 import * as actions from './actions';
 
 import type { Dataset } from '../../../types';
+import { searchDatasetRegistration, searchFullTextApi } from './operations';
+import { RegistrationStatus } from '../../../types/enums';
 
 function* searchRequested({
   payload: { searchRequest }
 }: ReturnType<typeof actions.searchDatasetsRequested>) {
   try {
     const auth = yield call(authService.getAuthorizationHeader);
 
-    const { data } = yield call(
-      axios.post,
-      `${getConfig().registrationApi.host}/search`,
-      searchRequest,
-      { headers: { Authorization: auth } }
+    const [registrationSearch, fulltextSearch] = yield all([
+      call(searchDatasetRegistration, searchRequest, auth),
+      call(searchFullTextApi, searchRequest)
+    ]);
+
+    const internalDatasetUris = new Set(
+      registrationSearch.map((dataset: Dataset) => dataset.uri)
+    );
```

Comment:
Det hadde holdt med:

```suggestion
    const internalDatasetUris = registrationSearch.map(({ uri }: Dataset) => uri)
```